.github/workflows/python-package.yml

```diff
@@ -33,8 +33,21 @@ jobs:
         # stop the build if there are Python syntax errors or undefined names
         flake8 climt/ --count --show-source --statistics
     - name: Build climt
-      env:
-        CLIMT_ARCH: Linux
       run: |
-        python setup.py build
-
+           if [ "$RUNNER_OS" == "Linux" ]; then
+            CLIMT_ARCH= Linux
```

Comment:
You cannot leave space during assignment of env variables in bash.
`CLIMT_ARCH= Linux` should be `CLIMT_ARCH=Linux`

Reply:
ok